testsuite/ui/views/admin/audience/developer_portal/__init__.py

```diff
@@ -34,6 +36,10 @@ def upload_logo(self, file):
         """Method choose logo and uploads it"""
         self.file_input.fill(file)
         self.upload_button.click()
+        try:
```

Comment:
Shouldn't this be an assertion in the actual test?

Reply:
"The uploaded Logo is not displayed correctly."